edge-apps/countdown-timer/screenly.yml

```
@@ -9,26 +9,26 @@ settings:
   countdown_headline:
     type: string
     default_value: New Year Countdown
-    title: countdown_headline
+    title: Countdown Headline Text
     optional: false
     help_text: Countdown header text value.
   enable_analytics:
     type: string
     default_value: 'true'
-    title: enable_analytics
+    title: Enable Analytics
     optional: true
     help_text: Whether to disable Sentry and Google Analytics integrations
     is_global: true
   override_locale:
     type: string
     default_value: en
-    title: override_locale
+    title: Override locale
```

```suggestion
    title: Override Locale
```

for being consistent